.appveyor.yml

```diff
@@ -18,4 +18,4 @@ test_script:
 build: off
 
 cache:
-  - node_modules
+  - node_modules -> package-lock.json
```

Comment:
What does the package-lock.json means here?

Reply:
It's how that cache gets invalidated, I was having trouble with this PR in appveyour and had to invalidate it properly.